src/views/PatronsPreRegistrationListContainer/PatronsPreRegistrationList.js

```diff
@@ -33,7 +35,7 @@ const PatronsPreRegistrationList = ({
   const renderActionColumn = (user) => (
     <Button
       type="button"
-      disabled={isLoading}
+      disabled={isLoading || !stripes.hasPerm('ui-users.patron-pre-registrations.execute')}
```

Comment:
UX pattern says that a non-permitted action should be hidden, not disabled: https://ux.folio.org/docs/guidelines/ux-patterns/hiding-vs-disabling-elements-ux-pattern/

Reply:
It is to the discretion of our PO. I have requested to review this comment.

Reply:
This is the response from our PO
![image](https://github.com/user-attachments/assets/6e6aa99d-0c4a-485f-8997-2aa28edb0fff)

Reply:
according to DoD we should follow UX guideline, it clearly says that action should be hidden as user can't do anything to enable an action (it's 100% a case for permissions as users usually can grant permissions themself)

hide is 1 line of code so I don't see any trouble to add it

Reply:
Hey @NikitaSedyx 
Please look into what our PO stated.

Reply:
your PO says `let's step back and follow FOLIO UX guidelines`

Reply:
Please re-read the statement. Feel free to contact our PO Irina Poklets for any further concerns.

Reply:
it's not my responsibility to talk to your PO about requirements, there is DoD and UX guideline,  your PO clearly stated to follow UX guideline